pyphare/pyphare/pharein/simulation.py

```
@@ -534,6 +534,23 @@ def check_restart_options(**kwargs):
     return restart_options
 
 
+def check_init_options(**kwargs):
+    """Advanced options to initialize from SAMRAI HDF5 files"""
+
+    formats = ["samraih5"]
+    init_options = kwargs.get("init_options", None)
+
+    if init_options is not None and "format" in init_options:
+        if init_options["format"] not in formats:
+            raise ValueError("Error - init_options format is invalid")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)